packages/edit-site/src/components/left-sidebar/navigation-panel/constants.js

```diff
@@ -0,0 +1,64 @@
+export const TEMPLATES_DEFAULT_DETAILS = {
+	// General
+	'front-page': {
+		title: 'Front page',
+		description: '',
```

Comment:
We could've added "Display the contents of your Front Page" here, or something along the similar lines.

Reply:
@david-szabo97 @vindl — Let's not forget i18n. Constants files like this should be i18n-ready from the start, otherwise getting everything ready for a testable Site Editor around WP 5.6 will be much more difficult.

Reply:
@mcsf My fault, I should have caught this in review!
I'll spin up a PR ASAP.